doc/Findr.txt

```diff
@@ -41,14 +41,20 @@ local to findr buffers
 General maps ~
                                                   *findr_<tab>*
                                                   *findr_<cr>*
+                                                  *findr_<c-x>*
+                                                  *findr_<c-v>*
+                                                  *findr_<c-t>*
 <cr>                      Choose selected item
 <tab>
+<c-x>                       in new split
+<c-v>                       in new vsplit
+<c-t>                       in new tab
```

Comment:
I'm not 100% positive this is how the docs should be laid out, but I was following the examples you had set out.

Reply:
I think that looks good